auslib/web/admin/views/required_signoffs.py

```diff
@@ -113,6 +113,13 @@ class ProductRequiredSignoffsView(RequiredSignoffsView):
     def __init__(self):
         super(ProductRequiredSignoffsView, self).__init__(dbo.productRequiredSignoffs, ["product", "channel", "role"])
 
+    def get(self):
+        where = {}
```

Comment:
Nit: A dict comprehension could be used here

```suggestion
        where = {
            param: request.args[param] for param in ("product", "channel") if param in request.args
        }
```

Similar changes can be made below.